atomic_reactor/plugins/pre_resolve_module_compose.py

```diff
@@ -137,9 +142,25 @@ def _resolve_compose(self):
         # since we our interaction with the PDC is read-only, this is fine for our needs and
         # makes things simpler.
         pdc_client = PDCClient(server=self.pdc_url, ssl_verify=not self.pdc_insecure, develop=True)
+        self.read_configs_general()
+
+        modules = self.data.get('modules', [])
+        if modules:
+            source_spec = modules[0]
+            if len(modules) > 1:
+                self.log.info("compose config contains multiple modules,"
+                              "using first module %s", source_spec)
+        else:
+            raise RuntimeError('"compose" config is missing "modules", required for Flatpak')
+
+        mod_name, mod_stream, mod_version = split_module_spec(source_spec)
```

Comment:
Can we call these `module_name`, `module_stream` etc for clarity (as in the original code)?

Reply:
ok as discussed will rename these, and loop bellow with conflicting names, will put into its own method